components/python/pytest/pytest-PYVER.p5m

```diff
@@ -95,12 +105,19 @@ file path=usr/lib/python$(PYVER)/vendor-packages/pytest.py
 #
 license pytest.license license=MIT
 
+# force a dependency on the Python runtime
+depend type=require fmri=__TBD pkg.debug.depend.file=python$(PYVER) \
+    pkg.debug.depend.path=usr/bin
+
 # force a dependency on argparse; pkgdepend work is needed to flush this out.
 # depend type=require fmri=library/python/argparse-$(PYV)
 
 # force a dependency on py; pkgdepend work is needed to flush this out.
 depend type=require fmri=library/python/py-$(PYV)
 
+# force a dependency on py; pkgdepend work is needed to flush this out.
+depend type=require fmri=library/python/pluggy-$(PYV)
```

Comment:
Comment says "py" while the code says "pluggy"

Reply:
Right :) Copy/paste :)

Reply:
Edited directly, I will squash and merge then.